### PR TITLE
Integration of HIBS simulations into main development branch

### PR DESCRIPTION
**Pull request by brunohcfaria**
This PR aims to add Luciano's contributions to the development branch and for releasing the first release of SHARC.

### CODE REVIEW

File: sharc/.vscode/settings.json
Comment (brunohcfaria) [inline sharc/.vscode/settings.json:1]: Shouldn't have this parameter file in the repository

File: sharc/.idea/.gitignore
Comment (brunohcfaria) [inline sharc/.idea/.gitignore:1]: Shouldn't have this parameter file in the repository

File: sharc/.idea/.name
Comment (brunohcfaria) [inline sharc/.idea/.name:1]: Shouldn't have this parameter file in the repository

File: sharc/.idea/misc.xml
Comment (brunohcfaria) [inline sharc/.idea/misc.xml:1]: Shouldn't have this parameter file in the repository

File: sharc/.idea/modules.xml
Comment (brunohcfaria) [inline sharc/.idea/modules.xml:1]: Shouldn't have this parameter file in the repository

File: sharc/.idea/sharc.iml
Comment (brunohcfaria) [inline sharc/.idea/sharc.iml:1]: Shouldn't have this parameter file in the repository

File: sharc/.idea/vcs.xml
Comment (brunohcfaria) [inline sharc/.idea/vcs.xml:1]: Shouldn't have this parameter file in the repository

**Review by LeoAguayoUnB — COMMENTED** [review]
Ok to merge this code into development branch, but there will be further changes in Antenna classes related to ITU-R M.1851-2.

**Comment by brunohcfaria** [comment]
We're closing this PR  because it was decided to add HIBS modifications incrementally to the simulator.

**Pull request closed without merging**